politeiad/backend/gitbe/decred.go

```diff
@@ -1408,11 +1353,7 @@ func (g *gitBackEnd) replayComments(token string) (map[string]decredplugin.Comme
 	}()
 
 	comments := make(map[string]decredplugin.Comment)
-
-	seen := make(map[string]map[string]int64)
-	if decredPluginCommentsUserCache != nil {
-		seen = decredPluginCommentsUserCache
-	}
+	commentsLikes := []decredplugin.LikeComment{}
```

Comment:
Please use make here with the somewhat correct length.

Reply:
I can make use of make here but It's not possible to know the correct length because the journals haven't been replayed at this point.

Reply:
Then use a PNOOMA.

Reply:
From slack resolution:
```
moo31337 [3:12 PM]
because that requires a bunch of mallocs under the hood

if you have no idea about size pick something large like 1024
make(x,0,1024)
if you do know the size do: make(x,0,len(xxx))
```